manager/system_handler.py

```diff
@@ -378,22 +388,23 @@ def handle_get(cls, **kwargs):  # pylint: disable=too-many-statements
         cves_view = SystemCvesView(list_arguments, {"rh_account_number": connexion.context['user'],
                                                     'inventory_id': inventory_id}, args, args, connexion.request.path, cls._ids_only)
 
+        cve_names = [cve['cve_name'] for cve in cves_view]
```

Comment:
don't iterate through cves_view more than once in this function, see https://github.com/RedHatInsights/vulnerability-engine/commit/fe137b67f937a8aa4989754f43b4a17e5bedf2f2